app/components/shared/notification_component_preview/preview.html.slim

```diff
@@ -0,0 +1,61 @@
+= form_for @preview_form, url: "/components/shared/notification_component/default", html: { method: "get" } do |f|
+  .govuk-grid-column-three-quarters
+
+    h1.govuk-heading-l = t("heading.component.notification.title")
+    p.govuk-body = t("heading.component.notification.description")
+
+    h2.govuk-heading-m = t("heading.options")
+    ul.govuk-list
+      li = t("heading.component.notification.options")
+
+    h2.govuk-heading-m = t("heading.examples")
+
+    - variants.each do |variant|
+      h3.govuk-heading-s = variant.title
+      p.govuk-body = variant.description
+      = render(Shared::NotificationComponent.new(icon: @preview_form.icon, title: @preview_form.title, body: @preview_form.body, variant: variant.id, background: @preview_form.background, dismiss: @preview_form.dismiss))
+
+      .govuk-tabs data={ "module": "govuk-tabs" }
+        ul.govuk-tabs__list
+          li.govuk-tabs__list-item.govuk-tabs__list-item--selected
+            a.govuk-tabs__tab href="#slim"
+              = t("heading.templates.slim")
+
+        .govuk-tabs__panel id="slim"
+          table.govuk-table
+            tbody.govuk-table__body
+              tr.govuk-table__row
+                td.govuk-table__cell
+                  pre
+                    code = "render(Shared::NotificationComponent.new(\n\t\ttitle: '#{@preview_form.title}',\n\t\tbody: '#{@preview_form.body}'\n\t},\n\tvariant: '#{@preview_form.variant}',\n\tbackground: #{@preview_form.background},\n\tdismiss: #{@preview_form.dismiss}\n))"
```

Comment:
Not sure if there is a nice way to do multiline strings in ruby in slim

Reply:
will be nicer way to do it it im sure, its a bit hardcoded for the purposes of this first preview - for the moment